bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -150,37 +150,46 @@ public ImageImpl() {
      */
     @PostConstruct
     protected void initModel() {
-        initFeaturedImageBasedProperties();
+        link = Optional.empty();
+        fileResource = resource.getChild(DownloadResource.NN_FILE);
+        initInheritedResource();
+        if (inheritedResource == null) {
+            return;
+        }
+        ValueMap inheritedResourceProperties = inheritedResource.getValueMap();
+        Resource inheritedFileResource = inheritedResource.getChild(DownloadResource.NN_FILE);
+        String inheritedFileReference = inheritedResourceProperties.get(DownloadResource.PN_REFERENCE, String.class);
```

Comment:
Do we still need to have both `fileReference` and `inheritedFileReference`? I would rather we overwrite `fileReference`. Same for `fileResource`.

Reply:
we need both as some properties are retrieved from the image resource (e.g. the `src` attribute), some are retrieved from the featured image resource.